resources/views/livewire/questions/create.blade.php

```diff
@@ -10,6 +10,7 @@ class="mb-12 pt-4"
         x-init='() => {
             uploadLimit = {{ $this->uploadLimit }};
             maxFileSize = {{ $this->maxFileSize }};
+            maxContentLength = {{ $this->maxContentLength }};
```

Comment:
how about? 
```js

x-data="imageUpload({{ $this->uploadLimit }},{{ $this->maxFileSize }},{{ $this->maxContentLength }})"
  
//  in js
  
const imageUpload = (uploadLimit, maxFileSize , maxContentLength) => ({});

Reply:
It might just be an extra abstraction, but doesn't hurt anything, so I'll add it for you.